### PR TITLE
Implement all clusters cc13x2x7

### PR DESCRIPTION
**Pull request by adabreuti**
#### Problem
Implements #17264 

#### Change overview
Add TI based all clusters implementation for CC2652R7 platform.
Added support to existing github workflow to build the example and report the bloat change

#### Testing
* Built application locally
* Verified base device commissioning on new application
* Verified onoff, levelcontrol, colorcontrol and identify clusters were able to write correct states from RPI chip-tool and verified states were correctly updated on Launchpad via the LED's and UART console output.

### CODE REVIEW

File: scripts/build/builders/cc13x2x7_26x2x7.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def ExampleName(self):
             return 'pump-app'
         elif self == cc13x2x7_26x2x7App.PUMP_CONTROLLER:
             return 'pump-controller-app'
+        elif self == cc13x2x7_26x2x7App.ALL_CLUSTERS:
```
Comment (andy31415) [inline scripts/build/builders/cc13x2x7_26x2x7.py:33]: missing ALL_CLUSTERS constant define

Comment (adabreuti) [reply scripts/build/builders/cc13x2x7_26x2x7.py:34]: Resolved, thanks

**Comment by adabreuti** [comment]
@bzbarsky-apple It looks like the Re-styled check is failing due to a GIT permission issue. Are you aware of any steps I can take to resolve this problem?

**Comment by bzbarsky-apple** [comment]
@adabreuti You can run restyle locally.  Simplest thing is to create a file called `restyle.sh` that looks like this:
```
: "${RESTYLER_TAG:=:master-final}"
exec docker run -it --rm \
  --env DEBUG=1 \
  --env HOST_DIRECTORY="$PWD" \
  --env UNRESTRICTED=1 \
  --volume "$PWD":/code \
  --volume /tmp:/tmp \
  --volume /var/run/docker.sock:/var/run/docker.sock \
  --entrypoint restyle-path \
  "restyled/restyler$RESTYLER_TAG" "$@"
```
and then run `restyle.sh` and pass it the filenames of all the files changes in this PR.  Then add the resulting changes to this PR.

**Comment by adabreuti** [comment]
> @adabreuti You can run restyle locally. Simplest thing is to create a file called `restyle.sh` that looks like this:
> 
> ```
> : "${RESTYLER_TAG:=:master-final}"
> exec docker run -it --rm \
>   --env DEBUG=1 \
>   --env HOST_DIRECTORY="$PWD" \
>   --env UNRESTRICTED=1 \
>   --volume "$PWD":/code \
>   --volume /tmp:/tmp \
>   --volume /var/run/docker.sock:/var/run/docker.sock \
>   --entrypoint restyle-path \
>   "restyled/restyler$RESTYLER_TAG" "$@"
> ```
> 
> and then run `restyle.sh` and pass it the filenames of all the files changes in this PR. Then add the resulting changes to this PR.

Got it, applied the changes. Thanks!

**Comment by bzbarsky-apple** [comment]
fast-tracking platform-specific change.

**Pull request merged**